client/src/app/(modules)/network/network-list.tsx

```diff
@@ -31,17 +47,31 @@ export default function NetworkList({
           No results based on your search criteria
         </p>
       )}
-      {networks?.map((g) => {
-        return (
-          <Network
-            key={`network-${g.type}-${g.id}`}
-            {...(g as Required<
-              ProjectListResponseDataItem &
-                OrganizationListResponseDataItem & { type: 'project' | 'organization' }
-            >)}
-          />
-        );
-      })}
+      <ul>
+        {networks?.map((g) => {
+          return (
+            <Network
+              key={`network-${g.type}-${g.id}`}
+              {...(g as Required<
+                ProjectListResponseDataItem &
+                  OrganizationListResponseDataItem & { type: 'project' | 'organization' }
+              >)}
+            />
+          );
+        })}
+      </ul>
+      <Button
+        ref={ref}
+        onClick={() => fetchNextPage()}
+        disabled={!hasNextPage || isFetchingNextPage}
+        className="mx-auto mt-8 flex"
+      >
+        {isFetchingNextPage
+          ? 'Loading more...'
+          : hasNextPage
+          ? 'Load more'
+          : 'Nothing more to load'}
```

Comment:
We don't need the button. We don't have design for this but I can replace it by a loader or something.

I'll update this list and the Datasets one as well because we're using the same elements.